manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```
@@ -76,7 +76,8 @@ spec:
             {{- end }}
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
-            - "--provider-volume=C:\\k\\secrets-store-csi-providers"
+            - "--provider-volume={{ .Values.windows.providersDir }}"            
+            - '--additional-provider-volume-paths={{ join  "," .Values.windows.additionalProvidersDirs }}'
```

```suggestion
            - "--additional-provider-volume-paths={{ join  "," .Values.windows.additionalProvidersDirs }}"
```

for consistency across all the flags configured in `args`

---

Ya so funny story, this will break the templating:

```
==> Linting manifest_staging/charts/secrets-store-csi-driver
[ERROR] templates/: template: secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml:80:60: executing "secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml" at <','>: expected string; found ','

Error: 1 chart(s) linted, 1 chart(s) failed
make: *** [lint-charts] Error 1
```

```
- "--additional-provider-volume-paths={{ join ',' .Values.linux.additionalProvidersDirs }}"
```

ALSO breaks the templating:

```
==> Linting manifest_staging/charts/secrets-store-csi-driver
[ERROR] templates/: template: secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml:80:58: executing "secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml" at <','>: expected string; found ','

Error: 1 chart(s) linted, 1 chart(s) failed
make: *** [lint-charts] Error 1
```

---

wait lemme try that again...